File: org.scala-ide.sdt.core.tests/test-workspace/simple-structure-builder/src/generic_signature/akka/Actor.java
package generic_signature.akka;

public class Actor {
	public static ActorRef actorOf2(Class<? extends Actor> clazz) {
		return new Actor2().actorOf(clazz); 
	}
	public static ActorRef actorOf3(Class<? extends Actor> clazz) {
		return Actor3.actorOf(clazz); 
	}
}

File: org.scala-ide.sdt.core.tests/test-workspace/simple-structure-builder/src/generic_signature/akka/Actor.scala
package generic_signature.akka

class ActorRef

class Actor2 {
  def actorOf(clazz: Class[_ <: Actor]): ActorRef = new ActorRef
}


object Actor3 {
  def actorOf(clazz: Class[_ <: Actor]): ActorRef = new ActorRef
}